src/modules/35-connectors/components/ConnectorReferenceField/FormMultiTypeConnectorField.tsx

```diff
@@ -56,7 +56,7 @@ export interface MultiTypeConnectorFieldConfigureOptionsProps
 }
 export interface MultiTypeConnectorFieldProps extends Omit<ConnectorReferenceFieldProps, 'onChange'> {
   onChange?: ExpressionAndRuntimeTypeProps['onChange']
-  formik?: FormikContext<any>
+  formik?: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
  formik?: FormikContextType<unknown>
```